The Cray wrappers are not detected as 'gcc' here and 
therefore we need the fix below in order to get the correct compilation flags: "-Wl,-R" 

@author: Christopher Bignamini (CSCS)
@author: Guilherme Peretti-Pezzi (CSCS)

--- EasyBuildTesting/Python-2.7.9/Lib/distutils.orig/unixccompiler.py	2014-12-10 16:59:34.000000000 +0100
+++ EasyBuildTesting/Python-2.7.9/Lib/distutils/unixccompiler.py	2015-06-12 16:02:06.000000000 +0200
@@ -237,7 +237,7 @@
         elif self._is_gcc(compiler):
             return "-Wl,-R" + dir
         else:
-            return "-R" + dir
+            return "-Wl,-R " + dir
 
     def library_option(self, lib):
         return "-l" + lib
